actors/common/src/main/java/org/sunbird/metrics/actors/CourseMetricsActor.java

```diff
@@ -174,6 +175,18 @@ private void courseProgressMetricsV2(Request actorMessage) {
     sender().tell(response, self());
   }
 
+  private void formatEnrolledOn(Map<String, Object> batchMap) {
+    String timeStamp = (String) batchMap.get(JsonKey.LAST_ACCESSED_ON);
+    try {
+      SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
```

Comment:
should be in some constants file?

And are we not supposed to read this format from some setting?